tools/lock_inspect/evm_connection.rb

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env ruby
+
+require 'pg'
+require 'yaml'
+require 'optparse'
+
+def error_exit(e, conn, error_message = '', exit_code = 1)
+  puts e.message
+  puts error_message unless error_message.empty?
+  conn.close if conn
+  exit(exit_code)
+end
+
+def process_app_name(app_name)
+  if app_name.end_with?('..')
+    raise "The application name for MIQ worker: {#app_name} is truncated"
+  end
+  items = app_name.split(/[\[\]\s,]+/)
```

Comment:
Thank you. This importance sequence is quite useful information. Currently parsing is not very difficult, as you can see, split base on brackets, commas and space, so the format is OK but I may need to change the sequence. @yrudman has another suggestion on this name, is this possible? To make class_name just a index and use a common hash or yml file to map class names. As in this
https://www.pivotaltracker.com/n/projects/1608513/stories/147871949
what cause truncated is always over long class name.